.github/workflows/simulation-1.2.yml

```diff
@@ -93,3 +93,49 @@ jobs:
     - name: Codecov
       run: |
         bash <(curl -s https://codecov.io/bash) -x "${{ matrix.compiler.gcov }}" -Z
+
+  thread-1-2-backbone:
+    runs-on: ubuntu-18.04
+    env:
+      REFERENCE_DEVICE: 1
+      VIRTUAL_TIME: 0
+      PACKET_VERIFICATION: 1
+      THREAD_VERSION: 1.2
+      INTER_OP: 1
+      COVERAGE: 1
+      MULTIPLY: 3
```

Comment:
Explain: run each test in `thread-cert/backbone/` directory for 3 times.

Reply:
Yes, we are being strict so that we can catch random verification fails with a higher chance. 

Note that current 1.1 packet-verification needs to pass 10 times, which helped (and urged) us to fix fails in 5.5.2 (https://github.com/openthread/openthread/pull/5490) and 6.1.9 (https://github.com/openthread/openthread/pull/5486) random fails. 

I think 10 times might be too harsh in a long term and 3 times can be a moderate choice.